assets/recipient-form.js

```diff
@@ -1,164 +1,162 @@
 if (!customElements.get('recipient-form')) {
-  customElements.define('recipient-form', class RecipientForm extends HTMLElement {
-    constructor() {
-      super();
-      this.checkboxInput = this.querySelector(`#Recipient-checkbox-${ this.dataset.sectionId }`);
-      this.checkboxInput.disabled = false;
-      this.hiddenControlField = this.querySelector(`#Recipient-control-${ this.dataset.sectionId }`);
-      this.hiddenControlField.disabled = true;
-      this.emailInput = this.querySelector(`#Recipient-email-${ this.dataset.sectionId }`);
-      this.nameInput = this.querySelector(`#Recipient-name-${ this.dataset.sectionId }`);
-      this.messageInput = this.querySelector(`#Recipient-message-${ this.dataset.sectionId }`);
-      this.sendonInput = this.querySelector(`#Recipient-send-on-${ this.dataset.sectionId }`);
-      this.offsetProperty = this.querySelector(`#Recipient-timezone-offset-${ this.dataset.sectionId }`);
-      if (this.offsetProperty) this.offsetProperty.value = new Date().getTimezoneOffset();
-
-      this.errorMessageWrapper = this.querySelector('.product-form__recipient-error-message-wrapper');
-      this.errorMessageList = this.errorMessageWrapper?.querySelector('ul');
-      this.errorMessage = this.errorMessageWrapper?.querySelector('.error-message');
-      this.defaultErrorHeader = this.errorMessage?.innerText;
-      this.currentProductVariantId = this.dataset.productVariantId;
-      this.addEventListener('change', this.onChange.bind(this));
-      this.onChange();
-    }
+  customElements.define(
+    'recipient-form',
+    class RecipientForm extends HTMLElement {
+      constructor() {
+        super();
+        this.checkboxInput = this.querySelector(`#Recipient-checkbox-${this.dataset.sectionId}`);
+        this.checkboxInput.disabled = false;
+        this.hiddenControlField = this.querySelector(`#Recipient-control-${this.dataset.sectionId}`);
+        this.hiddenControlField.disabled = true;
+        this.emailInput = this.querySelector(`#Recipient-email-${this.dataset.sectionId}`);
+        this.nameInput = this.querySelector(`#Recipient-name-${this.dataset.sectionId}`);
+        this.messageInput = this.querySelector(`#Recipient-message-${this.dataset.sectionId}`);
+        this.sendonInput = this.querySelector(`#Recipient-send-on-${this.dataset.sectionId}`);
+        this.offsetProperty = this.querySelector(`#Recipient-timezone-offset-${this.dataset.sectionId}`);
+        if (this.offsetProperty) this.offsetProperty.value = new Date().getTimezoneOffset().toString();
```

Comment:
Original comment from @fredma : 
> The backend already casts the property to int if its a string, so this is safe.

Reply:
@ludoboludo I think the original change was only this line, but the whole file seems to be changed 🤔 

https://github.com/Shopify/dawn/pull/2633/files#diff-b155c49124449ce256c03e05ec95dc95b232976c969766f023e30278c1833210L14

Reply:
Yeah formatting happened. I don't think Fred was using the prettier settings we do. So if you hide the whitespace while reviewing there should not be any change affecting the rest of the code: 
![](https://screenshot.click/12-27-u9a94-2lh9w.png)

Reply:
I still see a lot of other lines being affected. Is that expected?

Reply:
Can you share a screenshot of what you're seeing ? 
From my side it's just different formatting, spacing, semi colons, parentheses, etc. And that change with `.toString()`

Reply:
Sorry, what I mean is that since the file is already formatted in `main`, shouldnt the changes only be `toString`?

https://screenshot.click/12-09-6bevf-5f0j2.png

Reply:
I think those other changes are still just whitespace, it's just the GitHub whitespace filter not quite isolating them.

(interesting that prettier didn't do it's thing out of the box)

Reply:
> I dont see all the changes when I save it 🤔 but maybe i have different settings? (how customElements .define line went to a different line

When I prettify this file (outside this PR) I get the same type of [formatting](https://screenshot.click/12-13-2szle-uk8ew.mp4) seen here. I'm not 100% but I think prettier in its default state tends to break complex arguments like an anon function, or in this case a class definition, into their own lines even if the line was within our defined print width of 120. So this is expected as far as I can tell. 

Sofia, we can pair to see if/why your editor isn't using the same formatting as ours. Being a new addition, we've definitely had some cases of it not taking effect and instead using your existing editor/workspace preferences. Quitting vscode usually fixes it though. Either way, I would treat the formatting as a blocker.

Reply:
The formatting in `settings.json` should be like this: 
![](https://screenshot.click/12-15-n3m6q-kjc4p.png)

[Source](https://shopify.dev/docs/themes/tools/liquid-prettier-plugin#visual-studio-code)

Reply:
Ah! Mystery solved 😅 thanks Ludo! Definitely not a blocker - agreed